src/plasmapy/simulation/particle_tracker/particle_tracker.py

```diff
@@ -309,24 +329,39 @@ def _validate_constructor_inputs(
                 f"{field_weightings}",
             )
 
+    # Some quantities are necessary for the particle tracker to function
+    # regardless of other configurations
+    _REQUIRED_QUANTITIES: typing.ClassVar[set[str]] = {
+        "E_x",
+        "E_y",
+        "E_z",
+        "B_x",
+        "B_y",
+        "B_z",
+    }
+
     def _preprocess_grids(self, additional_required_quantities) -> None:
         """Add required quantities to grid objects.
 
-        Grids lacking the required quantities will be filled with zeros.
+        Grids lacking the default required quantities will be filled with zeros.
+        This method is not only called during instantiation of the |ParticleTracker|
+        but is also called when enabling stopping.
         """
 
-        # Some quantities are necessary for the particle tracker to function regardless of other configurations
-        required_quantities = {"E_x", "E_y", "E_z", "B_x", "B_y", "B_z"}
+        # Convert the required quantities set to a list to appease type hinting
+        quantities_list = list(self._REQUIRED_QUANTITIES)
 
         for grid in self.grids:
```

Comment:
So we should require that each quantity should be provided on at least one grid, otherwise an error will be emitted? What about simple cases where we only want to specify an E field, like in our Gaussian potential notebook?